app/models/scheduled_item.rb

```diff
@@ -1,9 +1,11 @@
 class ScheduledItem < ActiveRecord::Base
+  include Schedulable
+
   belongs_to :slide, touch: true
 
   mount_uploader :image, ImageUploader
 
-  include Schedulable
+  default_scope{ order(:order) }
```

Comment:
I've internalized the rule: don't use `default_scope`. Here's a StackOverflow questions addressing it:

- https://stackoverflow.com/questions/25087336/why-is-using-the-rails-default-scope-often-recommend-against

The tl;dr:

> There are multiple ways to overwrite the different effects of the default_scope. Getting that right gets complicated very quickly and I would argue not using the default_scope in the first place, would be a safer choice.

The alternative is to just be explicit about the scope you're calling (say, in the controller), which I think makes for more readable code.